docs/source/user_guide/installation/install-from-source.rst

```diff
@@ -304,36 +359,53 @@ Here are some additional useful commands you can run with ``Nox``:
 Troubleshooting
 ---------------
 
+**Problem:** I ran a ``nox``/python build command and encountered ``Could NOT find SUNDIALS (missing: SUNDIALS_INCLUDE_DIR SUNDIALS_LIBRARIES)`` error.
+
+**Solution:** This error occurs when the build system, ``scikit-build-core``, can not find the SUNDIALS libraries to build the ``IDAKLU`` solver.
+
+1. Run the following command to ensure SUNDIALS libraries are installed:
+
+   .. code:: bash
+
+		nox -s pybamm-requires -- --force
+
+2. If you are using a custom directory for SUNDIALS, set the ``INSTALL_DIR`` environment variable to specify the path:
+
+   .. code:: bash
+
+		export INSTALL_DIR=[custom_directory_path]
+
+**Problem:** When installing SUNDIALS, I encountered ``CMake Error: The source "../CMakeLists.txt" does not match the source "../CMakeLists.txt" used to generate cache`` error.
+
+**Solution:** This error occurs when there is a delay between installing and downloading SUNDIALS libraries.
+
+1. Remove the following directories from the PyBaMM directory if they exist:
+
+   a. ``download_KLU_Sundials``
+   b. ``sundials_KLU_libs``
+   c. Any custom directory you have set for installation
+
+2. Re-run the command to install SUNDIALS.
+3. If you are using a custom directory, make sure to set the ``INSTALL_DIR`` environment variable:
+
+   .. code:: bash
+
+		export INSTALL_DIR=[custom_directory_path]
+
 **Problem:** I have made edits to source files in PyBaMM, but these are
 not being used when I run my Python script.
 
-**Solution:** Make sure you have installed PyBaMM using the ``-e`` flag,
-i.e. ``pip install -e .``. This sets the installed location of the
-source files to your current directory.
-
-**Problem:** Errors when solving model
-``ValueError: Integrator name ida does not exist``, or
-``ValueError: Integrator name cvode does not exist``.
+**Solution:** Make sure you have installed PyBaMM using the ``-e`` flag, like so:
 
-**Solution:** This could mean that you have not installed
-``scikits.odes`` correctly, check the instructions given above and make
-sure each command was successful.
+.. code:: bash
 
-One possibility is that you have not set your ``LD_LIBRARY_PATH`` to
-point to the sundials library, type ``echo $LD_LIBRARY_PATH`` and make
-sure one of the directories printed out corresponds to where the
-SUNDIALS libraries are located.
+	pip install -ve .
 
-Another common reason is that you forget to install a BLAS library such
-as OpenBLAS before installing SUNDIALS. Check the cmake output when you
-configured SUNDIALS, it might say:
+If you want to install to contribute to PyBaMM, use this command:
```

Comment:
I meant that it would be nice for devs to opt for editable rebuilds over regular editable installations for "faster" development; at least, that was the idea. I should've been more clear here.